common/src/main/java/org/conscrypt/AddressUtils.java

```diff
@@ -56,7 +56,7 @@ public static boolean isValidSniHostname(String sniHostname) {
     /**
      * Returns true if the supplied hostname is an literal IP address.
      */
-    public static boolean isLiteralIpAddress(String hostname) {
+    static boolean isLiteralIpAddress(String hostname) {
```

Comment:
@kruton actually, I looked around and I'm not sure which Android tests use this.  Can you point me to it?